api/agentpb/agent.proto

```diff
@@ -329,6 +329,73 @@ message CheckConnectionResponse {
   Stats stats = 2;
 }
 
+// JobStatusRequest is a ServerMessage asking pmm-agent for job status.
+message JobStatusRequest {
+  string job_id = 1;
+}
+
+// JobStatusResponse is an AgentMessage containing job status.
+message JobStatusResponse {
+  bool alive = 1;
```

Comment:
maybe use enum? since we could have more statuses like stopped, stopped with error etc.

Reply:
If job stoped or terminated its result will be delivered to the server. We need this mostly for the case when pmm-agent crashed or restarted and we want to check if our job is alive.